htdocs/admin/company.php

```diff
@@ -431,33 +431,33 @@
 // Phone
 print '<tr class="oddeven"><td><label for="phone">'.$langs->trans("Phone").'</label></td><td>';
 print img_picto('', 'object_phoning', '', false, 0, 0, '', 'paddingright');
-print '<input name="tel" id="phone" value="'.dol_escape_htmltag($conf->global->MAIN_INFO_SOCIETE_TEL).'"></td></tr>';
+print '<input name="tel" id="phone" value="'.dol_escape_htmltag(!empty($conf->global->MAIN_INFO_SOCIETE_TEL) ? $conf->global->MAIN_INFO_SOCIETE_TEL : GETPOST("MAIN_INFO_SOCIETE_TEL", 'alphanohtml')).'"></td></tr>';
 print '</td></tr>'."\n";
 
 // Fax
 print '<tr class="oddeven"><td><label for="fax">'.$langs->trans("Fax").'</label></td><td>';
 print img_picto('', 'object_phoning_fax', '', false, 0, 0, '', 'paddingright');
-print '<input name="fax" id="fax" value="'.dol_escape_htmltag($conf->global->MAIN_INFO_SOCIETE_FAX).'"></td></tr>';
+print '<input name="fax" id="fax" value="'.dol_escape_htmltag(!empty($conf->global->MAIN_INFO_SOCIETE_FAX) ? $conf->global->MAIN_INFO_SOCIETE_FAX : GETPOST("MAIN_INFO_SOCIETE_FAX", 'alphanohtml')).'"></td></tr>';
 print '</td></tr>'."\n";
 
 // Email
 print '<tr class="oddeven"><td><label for="email">'.$langs->trans("EMail").'</label></td><td>';
 print img_picto('', 'object_email', '', false, 0, 0, '', 'paddingright');
-print '<input name="mail" id="email" class="minwidth200" value="'.dol_escape_htmltag($conf->global->MAIN_INFO_SOCIETE_MAIL).'"></td></tr>';
+print '<input name="mail" id="email" class="minwidth200" value="'.dol_escape_htmltag(!empty($conf->global->MAIN_INFO_SOCIETE_MAIL) ? $conf->global->MAIN_INFO_SOCIETE_MAIL : GETPOST("MAIN_INFO_SOCIETE_MAIL", 'alphanohtml')).'"></td></tr>';
 print '</td></tr>'."\n";
 
 // Web
 print '<tr class="oddeven"><td><label for="web">'.$langs->trans("Web").'</label></td><td>';
 print img_picto('', 'globe', '', false, 0, 0, '', 'paddingright');
-print '<input name="web" id="web" class="minwidth300" value="'.dol_escape_htmltag($conf->global->MAIN_INFO_SOCIETE_WEB).'"></td></tr>';
+print '<input name="web" id="web" class="minwidth300" value="'.dol_escape_htmltag(!empty($conf->global->MAIN_INFO_SOCIETE_WEB) ? $conf->global->MAIN_INFO_SOCIETE_WEB : GETPOST("MAIN_INFO_SOCIETE_WEB", 'alphanohtml')).'"></td></tr>';
```

Comment:
Same than previous comment. Here the GETPOST seems useless if MAIN_INFO_SOCIETE_WEB is already set and you submit form with error, the GETPOST will never be called

Reply:
@eldy it's fixed